src/scripts/destinyTrackerApi/itemListBuilder.js

```diff
@@ -0,0 +1,74 @@
+import _ from 'underscore';
+import { itemTransformer } from './itemTransformer.js';
+
+class itemListBuilder {
```

Comment:
I've gone with the underscore-is-private convention you suggested elsewhere, but the lack of access modifiers seems pretty Wat.